dev-docs/source/CondaPackageManager.rst

```diff
@@ -0,0 +1,108 @@
+.. _CondaPackageManager:
+
+=====================
+Conda Package Manager
+=====================
+
+.. contents::
+   :local:
+
+Mantid uses `Conda <https://docs.conda.io/en/latest/>`_ as its package management system. This document gives a
+developer overview on how we use the Conda package manager, including tips on how to debug dependency issues, and
+our policy towards using pip packages (it is strongly discouraged).
+
+Creating Environments
+---------------------
+
+Creating an empty environment:
```

Comment:
It might be good to be explicit here that the environment can be called anything.
```suggestion
Creating an empty environment called ``myenv``:
```